Sources/StreamChatUI/ChatMessageList/ChatMessageListView.swift

```diff
@@ -259,6 +262,28 @@ open class ChatMessageListView: UITableView, Customizable, ComponentsProvider {
         }
     }
 
+    private func getVisibleQuotedMessageUpdates(forUpdatedMessageAt indexPath: IndexPath) -> [IndexPath]? {
+        let visibleCells = getVisibleCells()
+        guard let visibleUpdatedMessage = visibleCells[indexPath]?.messageContentView?.content else {
+            return nil
+        }
+
+        var quotedMessageUpdates: [IndexPath] = []
+
+        for (index, messageCell) in visibleCells {
+            let quotedMessageId = messageCell.messageContentView?.content?.quotedMessage?.id
+            let visibleUpdatedMessageId = visibleUpdatedMessage.id
```

Comment:
You don't need to get `id` everytime, you can change the `guard` so it gets `visibleUpdatedMessageId` from the get go